packages/p2p/src/components/modal-manager/modals/create-ad-modal/__tests__/create-ad-modal.spec.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { screen, render, fireEvent } from '@testing-library/react';
+import CreateAdModal from '../create-ad-modal';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+
+const el_modal = document.createElement('div');
+
+// eslint-disable-next-line react/display-name
+jest.mock('../create-ad-modal-card', () => () => {
+    return <div data-testid='create-ad-modal-card' />;
```

Comment:
```suggestion
    return <div data-testid='dt_create_ad_modal_card' />;
```

We use this format for naming our test ids